### ISSUE TITLE
AMS initial database setup

### ISSUE THREAD

**Issue by ryzom-pipeline** [ryzom/ryzomcore#101]
**[Original report](https://bitbucket.org/ryzom/ryzomcore/issue/101) by Meelis Mägi (Bitbucket: [Meelis Mägi](https://bitbucket.org/Meelis Mägi), ).**

----------------------------------------

When AMS sets up initial database, it also creates 'admin' user for ams and the game.

However, it does not insert 'admin' user into `ring_open`.`ring_users` table which is required by client.

**Comment by ryzom-pipeline** [comment]
**Original comment by Matthew Lagoe (Bitbucket: [botanic](https://bitbucket.org/botanic), GitHub: [botanic](https://github.com/botanic)).**

----------------------------------------

are you using the latest ams code? there was a bug with the awile back but it should be fixed now

**Comment by ryzom-pipeline** [comment]
**Original comment by Meelis Mägi (Bitbucket: [Meelis Mägi](https://bitbucket.org/Meelis Mägi), ).**

----------------------------------------

It's from 'default' branch.

There is two mentions for `ring_users` table, first one creates the table and second reads cookie from it.

There is also $CREATE_RING config variable, that is never used.

Is that code merged to default?

**Comment by ryzom-pipeline** [comment]
**Original comment by Matthew Lagoe (Bitbucket: [botanic](https://bitbucket.org/botanic), GitHub: [botanic](https://github.com/botanic)).**

----------------------------------------

Humm i dont think i did anything with ring as ive never used it. Let me know what needs to be added and ill take care of it, you can also do a pull request or a patch.

**Comment by ryzom-pipeline** [comment]
**Original comment by Meelis Mägi (Bitbucket: [Meelis Mägi](https://bitbucket.org/Meelis Mägi), ).**

----------------------------------------

I have not played with AMS, so dont know where to start.

I'll just keep $AutoCreateRingInfo = true in ryzom/tools/server/www/login/config.php file for now.

**Comment by ryzom-pipeline** [comment]
**Original comment by Matthew Lagoe (Bitbucket: [botanic](https://bitbucket.org/botanic), GitHub: [botanic](https://github.com/botanic)).**

----------------------------------------

Just let me know what needs to be created for a user in those database's and ill add it

**Comment by ryzom-pipeline** [comment]
**Original comment by Meelis Mägi (Bitbucket: [Meelis Mägi](https://bitbucket.org/Meelis Mägi), ).**

----------------------------------------

r2_login.php does it this way (database is `ring_open`):
```
"INSERT INTO ring_users SET user_id = $id, user_name = '$login', user_type='ut_pioneer'"
```

**Comment by ryzom-pipeline** [comment]
**Original comment by Matthew Lagoe (Bitbucket: [botanic](https://bitbucket.org/botanic), GitHub: [botanic](https://github.com/botanic)).**

----------------------------------------

fixed in 8f4f45b
